src/EFCore.PG/Query/Internal/NpgsqlSqlTranslatingExpressionVisitor.cs

```diff
@@ -127,9 +128,11 @@ protected override Expression VisitMethodCall(MethodCallExpression methodCall)
             if (visited != null)
                 return visited;
 
-            // TODO: Handle List<>
-            if (methodCall.Arguments.Count > 0 && methodCall.Arguments[0].Type.IsArray)
+            if (methodCall.Arguments.Count > 0 && (
+                    methodCall.Arguments[0].Type.IsArray || methodCall.Arguments[0].Type.IsGenericList()))
```

Comment:
nit: Having all on a single line would be better, but it's up to you. Is it perf sensitive?

Reply:
I guess I'm starting to make more of an effort to respect 120 chars width :) Must be more bad EF influence...! But in this case it seems to express the logical structure better (or and and)...

> Is it perf sensitive?

You mean because of the double `methodCall.Arguments[0]`? Definitely not perf sensitive to that degree :) This is part of "compilation", which happens once per LINQ query structure and the results are then cached.